tests/functional/test_cluster_service_config_functions.py

```diff
@@ -84,7 +84,7 @@ def test_create_cluster_service_config(
         }
         _, cluster_svc = cluster_with_service
         with allure.step("Create config"):
-            _set_config(cluster_svc, cfg_json)
+            cluster_svc.config_set(cfg_json)
```

Comment:
there are many more places where this function is used. you can remove everything

Reply:
Some test cases require to set config directly because `config_set` at least expect iterable.
May be there is another workaround?